Signal/src/Models/TSMessageAdapaters/TSAnimatedAdapter.m

```diff
@@ -103,8 +103,7 @@ - (UIView *)mediaView {
         NSData *fileData = [NSData dataWithContentsOfURL:[self.attachment mediaURL]];
         if (!fileData) {
             DDLogError(@"%@ Could not load image: %@", [self tag], [self.attachment mediaURL]);
-            OWSAssert(0);
-            return nil;
+            return [UIView new];
```

Comment:
Hmm, maybe I'l convert these to have a gray background so they show up against the white background.